ketcher-autotests/tests/User-Interface/Clear-Canvas/clear-canvas.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { test } from '@playwright/test';
+import {
+  takeEditorScreenshot,
+  selectTopPanelButton,
+  delay,
+} from '@utils/canvas';
+import { clickInTheMiddleOfTheScreen, pressButton } from '@utils/clicks';
+import { waitForPageInit } from '@utils/common';
+import { openFileAndAddToCanvas } from '@utils/files';
+import { DELAY_IN_SECONDS } from '@utils/globals';
+import { RingButton, TopPanelButton, selectRing } from '@utils/selectors';
+import { addTextBoxToCanvas } from '@utils/selectors/addTextBoxToCanvas';
+
+test.describe('Clear canvas', () => {
+  test.beforeEach(async ({ page }) => {
+    await waitForPageInit(page);
+  });
+
+  test('Clear Canvas - "Clear canvas" button', async ({ page }) => {
+    // Test case:EPMLSOPKET-1702
+    // Checking apperance of the tooltip and deleting object with Clear canvas button
+    // await selectTopPanelButton(TopPanelButton.Clear, page);
+    await page.getByRole('button', { name: 'Clear Canvas (Ctrl+Del)' }).hover();
+    await page.getByTitle('Clear Canvas (Ctrl+Del)').click();
```

Comment:
You don't need to hover the mouse over the button. We already have a function to clear on Clear canvas button